### ISSUE TITLE
Exception after attempting to share photo: Database: net.sqlcipher.database.SQLiteException: file is not a database: , while compiling: select count(*) from sqlite_master;

### ISSUE THREAD

**Issue by doctaweeks** [signalapp/Signal-Android#8939]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
My partner attempted to share a photo from Google Photos through Signal and Signal crashed. Every time it is opened now it produces the same set of exceptions found in issue #8781. (I do not know what the original exception was; the phone was restarted already.) I've come to understand this means the DB is corrupt and (reasonably) unrecoverable. The device is not rooted.

This does not seem to be a spontaneous failure.

### Device info

**Device:** Motorola g7 power
**Android version:** 9 (March 1, 2019 patch)
**Signal version:** 4.43.8

**Comment by schmeat** [comment]
https://debuglogs.org/7e9dc2867105ea88baca5b24bff191be2bb0d16cc991fd59814cea50c80c4a87


I get a crash too whenever I try and open and image I sent.

**Comment by u32i64** [comment]
@schmeat As far as I see from your description

> try and open an~~d~~ image

and from your debug log:
```
E UncaughtExceptionLogger: java.lang.IllegalStateException: Fragment org.thoughtcrime.securesms.mediapreview.ImageMediaPreviewFragment must be a public static class to be properly recreated from instance state.
```
your issue is not related to the one you're commenting in and is actually a duplicate of #8940, which is fixed in Signal 4.44.1 (currently in beta, as was the 4.44.0 version which introduced the issue).
